hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/ratis/RatisHelper.java

```diff
@@ -269,23 +282,76 @@ static GrpcTlsConfig createTlsClientConfig(SecurityConfig conf,
     return tlsConfig;
   }
 
```

Comment:
can we add a table defining the exception no retry policy relationship for better understanding?